lib/queue_classic/conn_adapter.rb

```diff
@@ -55,6 +55,11 @@ def server_version
                           end
     end
 
+    def active?
```

Comment:
For reference: how activerecord does this check:

https://github.com/rails/rails/blob/cadc5de223c4de30cc8280dbb335d02b07c0c44f/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L262-L270

Reply:
I saw this, but didn't like the exception-driven execution path that this method encourages.
Digging through the code at the time, I felt that this was covering all the bases. But it has been too long for me to remember.